content/articles/auto-renew-only-domains.md

```diff
@@ -5,14 +5,18 @@ categories:
 - Domains
 ---
 
-# "Auto-Renew" Only Domains
+# Auto-Renew Only Domains
```

Comment:
I took the quotes out here, but was hesitant to remove them from the actual title as they serve the purpose of differentiating that the domains are auto-renew only in the search so people don't end up here vs in domain auto-renewal (I am very open to suggestions here re: the title)

Reply:
I totally get you. Here are some options: Auto-Renewal Mandatory Domains or Auto-Renewal Exclusive Domains, just brainstorming here.

Reply:
Mandatory Auto-Renewal Domains?

Reply:
I think Mandatory conveys this better 👍, thanks y'all

Reply:
I thought of 'Required' when I went to update to 'Mandatory', how does that look?